content/en/docs/tutorials/kubernetes-basics/_index.md

```diff
@@ -0,0 +1,56 @@
+---
+title: Learn Kubernetes Basics
+main_menu: true
+no_list: true
+weight: 20
+content_type: concept
+---
+
+## Kubernetes Basics
+
+This tutorial provides a walkthrough of the basics of the Kubernetes cluster orchestration
+system. Each module contains some background information on major Kubernetes features
+and concepts, and a tutorial for you to follow along.
+
+Using the tutorials, you can learn to:
+
+* Deploy a containerized application on a cluster.
+* Scale the deployment.
+* Update the containerized application with a new software version.
+* Debug the containerized application.
+
+## What can Kubernetes do for you?
+
+With modern web services, users expect applications to be available 24/7, and developers
+expect to deploy new versions of those applications several times a day. Containerization
+helps package software to serve these goals, enabling applications to be released and updated
+without downtime. Kubernetes helps you make sure those containerized applications run where
+and when you want, and helps them find the resources and tools they need to work. Kubernetes
+is a production-ready, open source platform designed with Google's accumulated experience in
+container orchestration, combined with best-of-breed ideas from the community.
+
+## Kubernetes Basic Modules
```

Comment:
I think this current sub-section [(preview here)](https://deploy-preview-48592--kubernetes-io-main-staging.netlify.app/docs/tutorials/kubernetes-basics/#kubernetes-basic-modules) of the list of tutorial pages feels a bit confusing with the links and images. It might be more effective to use a simple bullet list of links without images. Just my first thought looking at the [page](https://deploy-preview-48592--kubernetes-io-main-staging.netlify.app/docs/tutorials/kubernetes-basics/) - I’m sure others may have different opinions on this.

Reply:
Yes, I don't like it, either but I didn't know how much leeway I have to change the original content.

Reply:
I'll upload a draft with just text links